### ISSUE TITLE
[feat]

### ISSUE THREAD

**Issue by PeteRager** [zwave-js/zwavejs2mqtt#2357]
**Is your feature request related to a problem? Please describe.**
There seems to be large variance in how zwave devices behave.  There are decisions in this project regarding how they should behave.  The request is to allow configurable behaviors.  As a specific example, GE/Jasco In-Wall 3-Speed Fan Control 55258 / ZW4002.  When this device is commanded to Off, it reports back it’s current state (On) which then causes the “poll to be cancelled” and the current state remains on until a poll is done.  This is an example where the optimization to cancel the poll causes data accuracy issues in downstream apps.

**Describe the solution you'd like**
On a per node basis or per node CC basis, allow the poll after set to be configurable to allow disabling of the cancel and/or specification on what to do (poll immediately, poll after x ms delay). This would be the basis for allowing other configurable behaviors - like poll at startup, etc.  The minimal would to disable this poll cancel on per device. 

**Describe alternatives you've considered**
I have written an automation in HA to work around, it’s not graceful or fast or efficient,   Another viable approach could be to update the HA zwave js integrations fan entity to issue the poll if configured for the device, but this only fixes it for HA users.  The most unappealing alternative is to replace these switches.

**Additional context**

**Comment by robertsLando** [comment]
I would do such things with tools like Node-Red. Creating this sort of automations could make the configuration too complicated

**Comment by PeteRager** [comment]
Sounds interesting, point me to an example?

**Comment by robertsLando** [comment]
There are tons of examples/tutorials about how to use node-red. I would simply use MQTT to intercept node values and trigger node to schedule the pools. If you need a broker there is also a node that allows you to setup an mqtt broker

**Comment by PeteRager** [comment]
Updated my prior comment please review

**Comment by robertsLando** [comment]
> Would you consider a change to disable the poll optimization. Perhaps make that the default and then allow people to turn it on if needed?

I still don't understand what youy mean, try open a PR and I will check that on the code

**Comment by PeteRager** [comment]
I will try do that.  My JS skills are limited.

The code snippet above is from your current code base and part of this change:

https://github.com/zwave-js/node-zwave-js/pull/2410

That optimization has broken this type of device.  If I back out that change by hacking the function snippet above to return False all the time (e.g. disabled this optimization), then all is good.

**Comment by robertsLando** [comment]
@PeteRager That code is part of zwave-js not zwavejs2mqtt.

cc @AlCalzone

**Comment by AlCalzone** [comment]
> The fundamental issue is the poll cancel optimization breaks this device for every user

I disagree. The fundamental issue is that the device reports a different state than it is in, like you've described in the OP. Basically the device sends false data.

Nonetheless, disabling the poll-canceling seems like it has a broad enough benefit that we could add a compat flag for. There are other devices which only report intermediate dimming values for example, which could benefit from this aswell.
